### PR TITLE
Cassandra 3.x-only: drop type=ColumnFamily in favor of type=Table

### PR DESCRIPTION
**Pull request by eevans**
Bug: https://phabricator.wikimedia.org/T164093

### CODE REVIEW

**Comment by filippog** [comment]
👍

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.wikimedia</groupId>
   <artifactId>cassandra-metrics-collector</artifactId>
-  <version>3.1.4-SNAPSHOT</version>
+  <version>4.0.0-SNAPSHOT</version>
```
Comment (d00rman) [inline pom.xml]: Why such a _drastic_ version bump?

Comment (eevans) [inline pom.xml]: Because with this change, it's no longer backward compatible (no longer compatible w/ Cassandra < 3)

**Pull request merged**